Automattic/sensei
# Remove setup notices

Fixes #3046

### Changes proposed in this Pull Request

* Remove **Install Sensei LMS Pages** notice and related handlers, success notices. 
* Remove **Enable Usage Tracking** notice, dialog, handlers and tests.
* Deprecate public methods related to those

### Testing instructions

* Reset a site
* Activate Sensei LMS plugin
* No notices other than the Setup Wizard one should be visible.


### Deprecated Code

* `Sensei_Admin::install_pages_output`
* `Sensei_Admin::admin_install_notice`
* `Sensei_Admin::admin_installed_notice`
* `Sensei_Admin::admin_notices_styles`
* `Sensei_Admin::install_pages`

* `Sensei_Usage_Tracking_Base::hide_tracking_opt_in`
* `Sensei_Usage_Tracking_Base::is_opt_in_hidden`
* `Sensei_Usage_Tracking_Base::opt_in_dialog_text_allowed_html`
* `Sensei_Usage_Tracking_Base::maybe_display_tracking_opt_in`
* `Sensei_Usage_Tracking_Base::handle_tracking_opt_in`
* `Sensei_Usage_Tracking_Base::enqueue_script_deps`
* `Sensei_Usage_Tracking_Base::output_opt_in_js`
